python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -112,6 +112,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
     if missing_cred:
         raise Exception("Failed due to missing credentials %s" % str(missing_cred))
 
+    # Make sure that all the config options are really set.
+    # Some from plugins may not have taken during startup
```

Comment:
I'm concerned by this because to my mind we're using the correct config here from the bootstrap which doesn't exhibit this.
Is this due to restarting Ganga? In which case I'd say this should be in the else statement above (although yes, technically it doesn't cause bad things here)
Where exactly are the plugins loaded here again?